modules/rpc/handler.go

```diff
@@ -26,6 +26,7 @@ import (
 	"go.uber.org/fx"
 	"go.uber.org/fx/internal/fxcontext"
 	"go.uber.org/fx/service"
+	"go.uber.org/fx/uauth"
```

Comment:
I’m tempted to advise in the other direction—package names are “global” for the purposes of tools that automatically infer imports. I’d be tempted to prefix or suffix FX modules with fx to give tools a push, the way we have "yarpc/yarpctest" and not merely "yarpc/test".